core/server/proxy/src/main/java/alluxio/proxy/PathsRestServiceHandler.java

```diff
@@ -144,7 +141,7 @@ public Integer call() throws Exception {
         }
         return mStreamCache.put(os);
       }
-    }, ServerConfiguration.global());
+    } , ServerConfiguration.global(), null);
```

Comment:
nit: can you remove the unnecessary space in "} ,"?

Reply:
done